doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -20,20 +20,155 @@
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable storage. |
+| `req.inf.stg.06` | Storage | The Architecture **must** support ephemeral storage (non-persistent) storage for Pods. |
+| `req.inf.stg.07` | Storage | The Architecture **must** support persistent storage for Pods. |
+| `req.inf.ntw.01` | Network | The Architecture **must** support Conatiner Network Interface (CNI). |
+| `req.inf.ntw.02` | Network | The Architecture **must** support intra-node communications, such as between agents on a node and all pods on that node |
+| `req.inf.ntw.03` | Network | The Architecture **must** support inter-node communications without NAT, such as communications between pods on a node with all other pods on all nodes |
+| `req.inf.ntw.04` | Network | The Architecture **must** support low latency and high throughput traffic needs. |
+| `req.inf.ntw.05` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.06` | Network | The Architecture **must** support network resiliency. |
+| `req.inf.ntw.07` | Network | The Architecture **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.08` | Network | The Architecture **must** be fully redundant. |
+| `req.inf.ntw.09` | Network | The Architecture **should** support life-cycle management (LCM) of the infrastructure that allows repeatable, scalable and standardised handling of K8s clusters. LCM use cases to cover are: instantiation, termination, upgrade, scaling and healing. |
+| `req.inf.ntw.10` | Network | The networking solution **should** be configurable in an automated manner using openly published API definitions. |
+| `req.inf.ntw.11` | Network | The networking solution **should** be able to be centrally administrated and configured. |
+| `req.inf.ntw.12` | Network | The Architecture **must** support dual stack IPv4 and IPv6 for Kubernetes workloads. |
+| `req.inf.ntw.13` | Network | The Architecture **should** support IPv6 for Kubernetes workloads. |
+| `req.inf.ntw.14` | Network | The Architecture **must** support capabilities for integrating SDN controllers. |
+| `req.inf.ntw.15` | Network | The Architecture **should** support Service Mesh Interface (SMI). |
+| `req.inf.ntw.16` | Network | The Architecture **should**  support co-exisence of multiple Conatiner Network Interface (CNI). |
+| `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
+| `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
+| `req.inf.vir.01`   | Virtualisation |   The Architecture **must** support the capability for Containers to consume virtualised compute, storage and network resources.|
+| `req.inf.phy.01`  |  Physical Infrastructure |   The Architecture **must** support the capability for Containers to consume physical (aka bare metal) compute, storage and network resources. |
```

Comment:
@hdamker - my view is that the current wording allows for this, but happy to take suggestions.  I specifically used "must support the capability to" instead of "must run containers on" or something similar.  The intent (as per #562) is to identify the requirements that might exist for an underlying cloud in order for a Kubernetes RI to conform to RA2 and document them in RA2.  For example, we can list the things that need deploying and perhaps any specific configuration options for OpenStack, for AWS, packet, etc. Then each RI will (as you suggest) choose a cloud, and use the RA2 spec and guidance to create the RI for that Kubernetes+Cloud combination.  Does that make sense?

Reply:
@hdamker I've added these to the backlog (#606, #607) and they've been commented out for now.  Are you happy to resolve this conversation and mark the PR as approved as a result please?

Reply:
@tomkivlin First of all: I'm happy to resolve this conversation. I fully agree with your intent.

But one note which I don't know how to integrate into the two seperated issues: We should find a way that an RI does not need to comply with  req.inf.vir.01 and req.inf.phy.01 at the same time. An RI should be able to comply either to the one or the other. Maybe best to combine into one requirement?

Reply:
Yes agree - these are requirements for the RA though, not the RI :) But let's discuss in the related issues and gain consensus there. Thanks again for the help in clarifying.